AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -635,22 +635,15 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
   ASLayout *previousLayout = _layout;
   ASLayout *newLayout = [self calculateLayoutThatFits:constrainedSize];
   
-  if (ASHierarchyStateIncludesLayoutPending(_hierarchyState)) {
-    _pendingLayoutTransition = [[ASLayoutTransition alloc] initWithNode:self
-                                                          pendingLayout:newLayout
-                                                         previousLayout:previousLayout];
-  } else {
-    ASLayoutTransition *layoutTransition = nil;
-    if (self.usesImplicitHierarchyManagement) {
```

Comment:
@maicki could you tell me your thoughts here, I think this is OK because we want to enable it by default, but it's hard to tell if eliminating this condition will lead to the same behavior with today's implementation / client usage?